#Sensor values that autonomy need that is not part of perception data

int8 bcAttitudePortPot=0
int8 bcAttitudeStarboardPot=1
int8 depLoadCell=2
int8 depLowerLimit=3
int8 depUpperLimit=4
int8 excForeLimit=5
int8 excAftLimit=6
int8 bcLowerLimit=7
int8 bcUpperLimit=8
int8 sensorID
float32 value
